### PR TITLE
[CWS] Generate .length field for string fields

### PR DESCRIPTION
**Pull request by lebauce**
### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Comment by paulcacheux** [comment]
I'm not sure I understand why `len(container.id)` was not possible. This solution seems to add a lot of noise in the documentation and a lot of noise in the generation template

File: pkg/trace/go.mod [hunk 1]
```diff
@@ -70,7 +70,7 @@ require (
 	github.com/googleapis/gnostic v0.5.5 // indirect
 	github.com/hashicorp/hcl v1.0.0 // indirect
 	github.com/hectane/go-acl v0.0.0-20190604041725-da78bae5fc95 // indirect
-	github.com/imdario/mergo v0.3.12 // indirect
+	github.com/imdario/mergo v0.3.13 // indirect
```
Comment (paulcacheux) [inline pkg/trace/go.mod:73]: Why do we need this ?

**Review by maycmlee — COMMENTED** [review]
Just an edit for consistency.

**Review by maycmlee — APPROVED** [review]
👍 for docs

**Comment by paulcacheux** [comment]
Can you rebase, and restrict the length field on specific fields only

**Review by paulcacheux — APPROVED** [review]

**Pull request merged**